project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import sbt._
 object Dependencies {
 
   object Version {
-    val scala212 = "2.12.12"
+    val scala212 = "2.12.13"
```

Comment:
This update is blocked by an sbt-coverage issue (scoverage/sbt-scoverage#321). Is this required for this PR?

Reply:
Downgraded. Bumped it while was looking through code trying to understand the project. Didn't know that there are issues with coverage as used coverage only for Scala 2.13 before.